Loop.xcodeproj/xcshareddata/xcschemes/Complication - WatchApp.xcscheme

```diff
@@ -90,8 +90,8 @@
       launchAutomaticallySubstyle = "32">
       <RemoteRunnable
          runnableDebuggingMode = "2"
-         BundleIdentifier = "com.apple.carousel"
-         RemotePath = "/Naterade">
+         BundleIdentifier = "com.apple.Carousel"
+         RemotePath = "/$(MAIN_APP_DISPLAY_NAME)">
```

Comment:
Have you actually gotten the debugger to launch with the watch using this? I've been having problems lately with 11.7.

Reply:
In the iOS simulator or IRL?

Reply:
In the simulator.

Reply:
It seems to work the same as 11.7: namely, I can launch Loop in the iPhone simulator, and it works.  When I launch the Watch app in the debugger, it starts, but it "quits" the prior iPhone simulator run of Loop (11.7 did this for me too).  But the watch and iphone are stil linked, apparently.  Is that what you mean?

Reply:
Hm.. oddly, after testing it in the iOS simulator(s), it rewrote these scheme files (??).  I've posted updates